assets/src/amp-validation/amp-validation-detail-toggle.js

```diff
@@ -81,5 +81,5 @@ domReady( () => {
 			} );
 		} );
 
-	setValidationErrorRowsSeenClass();
```

Comment:
Seems this is no longer necessary as this function is only relevant to the AMP Validated URL screen.

Reply:
So after checking this one it turns out that it is used on the AMP Validation Error Index screen, too. What's more, the code also (incorrectly) interfered with the AMP Validated URLs screen (add  `new` class to every table row).

So, in 32afaf51f1e4bd1ec650241e4f05db85201395b6 I've updated the `setValidationErrorRowsSeenClass()` implementation so that the `new` class is toggled only if the hidden input is in the `column-status`. Since the hidden input is in the `.column-error_status` column on the AMP Validated URLs screen, it's not affected.

What's more, the `amp-validation-detail-toggle.js` is enqueued on every AMP admin page so I decided to keep it there and [remove](https://github.com/ampproject/amp-wp/pull/6067/commits/32afaf51f1e4bd1ec650241e4f05db85201395b6#diff-12c01fb0e2315d77441bb412562430cf3ecb4d5aeb493bf79440ca8640f78a54L32) it from the `amp-validated-url-post-edit-screen.js` instead.